Tasks/UseDotNetV2/globaljsonfetcher.ts

```diff
@@ -0,0 +1,69 @@
+import * as fileSystem from "fs";
```

Comment:
You should add unit tests for this file. You can reference version fetcher's unit tests.